tests/transport_test.go

```diff
@@ -2,42 +2,56 @@ package tests
 
 import (
 	"fmt"
+	"strconv"
+	"strings"
+	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
 	"github.com/pkg/errors"
 
-	"kubevirt.io/containerized-data-importer/tests/framework"
-	"kubevirt.io/containerized-data-importer/tests/utils"
-
 	"k8s.io/api/core/v1"
 	"kubevirt.io/containerized-data-importer/pkg/common"
 	"kubevirt.io/containerized-data-importer/pkg/controller"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
 )
 
 var _ = Describe("Transport Tests", func() {
 
 	const (
 		secretPrefix = "transport-e2e-sec"
 		targetFile   = "tinyCore.iso"
+		sizeCheckPod = "size-checker"
 	)
 
 	f, err := framework.NewFramework("", framework.Config{SkipNamespaceCreation: false})
 	handelError(errors.Wrap(err, "error creating test framework"))
 
-	c, err := f.GetKubeClient()
+	c := f.K8sClient
 	handelError(errors.Wrap(err, "error creating k8s client"))
 
+	// Use http and trim api port from url
+	host := strings.Replace(f.RestConfig.Host, "https", "http", 1)
```

Comment:
would the _url_ pkg help here?

Reply:
I thought it added a lot of overhead for a simple operation.  I'm not against using it if we think it's worth it though.

Reply:
that's fine...